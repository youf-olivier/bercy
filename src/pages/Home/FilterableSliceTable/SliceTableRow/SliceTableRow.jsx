import React from 'react';
import { Table } from '@axa-fr/react-toolkit-all';

export const SliceTableRow = ({ lowBorn, highBorn, rate }) => {
  return (
    <Table.Tr>
      <Table.Td>
        <span className="af-table-body-content">
          de {lowBorn}€ à {highBorn}€
        </span>
      </Table.Td>
      <Table.Td>
        <b>{rate}%</b>
      </Table.Td>
    </Table.Tr>
  );
};
