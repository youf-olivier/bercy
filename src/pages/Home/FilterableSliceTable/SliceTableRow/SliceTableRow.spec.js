import { render } from '@testing-library/react';
import { SliceTableHeader } from 'pages/Home/FilterableSliceTable/SliceTableHeader/SliceTableHeader';
import { SliceTableRow } from 'pages/Home/FilterableSliceTable/SliceTableRow/SliceTableRow';

describe('SliceTableRow tests suite', () => {
  it('SliceTableRow should render without crashing', () => {
    const { asFragment } = render(
      <SliceTableRow lowBorn="200" highBorn="50000" rate="2" />,
    );
    expect(asFragment).toMatchSnapshot();
  });
});
