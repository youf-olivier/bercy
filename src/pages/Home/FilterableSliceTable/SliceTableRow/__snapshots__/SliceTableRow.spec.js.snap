// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SliceTableRow tests suite SliceTableRow should render without crashing 1`] = `
<DocumentFragment>
  <span
    aria-label="tranche"
    class="af-table-body-content"
  >
    de 200€ à 50000€
  </span>
  <span
    aria-label="taux imposition"
  >
    <b>
      2%
    </b>
  </span>
</DocumentFragment>
`;
